File: c#/validatePin.cs
        public static bool ValidatePIN(string pin) 
        /* ATM machines allow 4 or 6 digit PIN codes and PIN codes cannot contain anything but exactly 4 digits or exactly 6 digits.
        
        Your task is to create a function that takes a string and returns true if the PIN is valid and false if it's not.*/

        {

            if (pin.Length != 4 && pin.Length != 6) {

                return false;

            }
            char[] charArray = pin.ToCharArray();

            foreach (char ch in charArray) {

                if (!char.IsLetter(ch)) {


                    return false;

                }


            

            }

            return true;
        }

File: python/validatePin.py
'''

Create a function to test if a string is a valid pin or not.

A valid pin has:

    Exactly 4 or 6 characters.
    Only numerical characters (0-9).
    No whitespace.

'''

def valid(txt):
    if len(txt)!=4 and len(txt)!=6:
        return False
    if not txt.isdigit():
        return False
    if " " in txt:
        return False
    return True
        
        


if __name__ == "__main__":
    print(valid("451356"))